COST-587: Update nise to generate new Azure export files.

myersCody opened the pull request:
I added a `-v2` flag to use the version two of the report.
- ` -v2, --version-two    Generate version two of the azure report.`


### How to test
```
nise yaml azure -o azure_test.yaml
make install && nise report azure -w --static-report-file azure_test.yaml -v2
```

I added the -w flag so that way the monthly files would be produced. Then I pull that csv up on my computer to compare the different headers and values between azure report version one and azure report version two.

What I have left to do:

- [x] Run one of these azure reports through the koku server to make sure its processed correctly.

----
### Smoke Tests
```===== 3957 passed, 80 skipped, 1736 deselected in 1884.97s (0:31:24) =====```

maskarb commented on nise/__init__.py:
```diff
@@ -1,2 +1,2 @@
-__version__ = "2.0.18"
```
lets make this 2.0.19

myersCody replied:
@maskarb 
So, I was going off of the versions of koku-nise in pypi: https://pypi.org/project/koku-nise/#history

I think when we merged the 2.0 branch back to master, things got a little wonky with what is in pypi and what the master version is. I think jumping back up to `2.1.6` will bring us back into sync.

maskarb replied:
In that case, let's jump 2.1, and go to 2.2.0

maskarb commented on nise/report.py:
```diff
@@ -69,6 +69,8 @@
 from nise.upload import upload_to_s3
 from nise.util import LOG
 
+pp = pprint.PrettyPrinter()
```
Don't see this used anywhere

myersCody commented:
How I tested if the koku server could read the new azure report version I generated here

1. First I brought up the koku pip environment, and then did a `pip uninstall koku-nise`.
1. Then I switched over to the nise repo with this branch pulled up and ran a `make install`
1. Then I modified the `load_test_customer_data.sh` and when we were generating the azure report, I added the `-v2 -w` parameters. So that way the monthly files would be written. 
1. After that I brought up a fresh version for the server, and ran `make docker-reinitdb-with-sources`
1. After that I ran `make load-test-customer-data` and hit the http://127.0.0.1:8000/api/cost-management/v1/reports/azure/costs/ endpoint to make sure it wasn't empty

Let me know if there is any other manual checking I should do besides this.
